src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -67,4 +81,22 @@ def add_cookie(self, name, value, path=None, domain=None, secure=None):
         # Secure should be True or False
         if is_truthy(secure):
             new_cookie['secure'] = secure
+        if is_truthy(expiry):
+            expiry_datetime = datetime.strptime(expiry, '%Y-%m-%d %H:%M:%S')
```

Comment:
It would be better to use Robot Framework DataTime library for better support different time formats. 

Example something like this could work: 
```
from robot.libraries.DateTime import convert_date

...

expiry_datetime = int(convert_date('20140528 12:05:03.111', result_format='epoch'))
```

Then the keyword documentation could say that `expiry` supports same formats as DateTime library,